README.md

```diff
@@ -4,15 +4,15 @@
 [![goto counter](https://img.shields.io/github/search/tbodt/ish/goto.svg)](https://github.com/tbodt/ish/search?q=goto)
 [![fuck counter](https://img.shields.io/github/search/tbodt/ish/fuck.svg)](https://github.com/tbodt/ish/search?q=fuck)
 
-A project to get a Linux shell running on iOS, using usermode x86 emulation and syscall translation.
+iSH is a Linux shell on iOS with usermode x86 emulation and syscall translation.
```

Comment:
I prefer the earlier wording:
```suggestion
iSH is a Linux shell running iOS, using usermode x86 emulation and syscall translation.
```

Reply:
Ah yeah, updated the suggestion.

Reply:
"Running on" seems implied to me but I'm okay with the extra clarity of specifying.

"With" captures how x86 emulation and syscall translation are features of iSH but "using" implies they are from another library.

Reply:
How about:
```suggestion
iSH is a Linux shell running on iOS done with usermode x86 emulation and syscall translation.```

Reply:
I'd prefer
```suggestion
iSH is a Linux shell on iOS featuring usermode x86 emulation and syscall translation.
```
or keeping it the same as before.

Reply:
"x86 emulation and syscall translation" isn't really a list of features, it's more of a description of how it's accomplished.

Reply:
Alright, "done" seems appropriate
```suggestion
iSH is a Linux shell running on iOS, done with usermode x86 emulation and syscall translation.
```